content/documentation/staging/configuration/authentication/openid.adoc

```diff
@@ -69,11 +75,17 @@ kubectl create secret generic kiali --from-literal="oidc-secret=$CLIENT_SECRET"
 
 where `$NAMESPACE` is the namespace where you installed Kiali and
 `$CLIENT_SECRET` is the secret you configured or provided by your OpenId
-Server.
+Server. If Kiali is already running, you may need to restart the Kiali pod so
+that the secret is mounted in Kiali.
 
-icon:bullhorn[size=1x]{nbsp} This secret is not needed if you don't want
-Kiali to use the _authorization code flow_ (i.e. if your signing key is neither 16,
-24 or 32 bytes long).
+icon:bullhorn[size=1x]{nbsp} This secret is not needed if you don't want Kiali
+to use the _authorization code flow_ (i.e. if your Kiali's signing key is
+neither 16, 24 or 32 bytes long).
+
+icon:bullhorn[size=1x]{nbsp} It's worth to empazhise that you need to re-use
```

Comment:
to empazhise --> emphasizing

Actually, maybe restructure the sentence like:

It's worth emphasizing that to configure OpenID integration you must re-use the OpenID application/client that you created for your Kubernetes cluster.